src/views/App/index.tsx

```diff
@@ -35,12 +36,16 @@ class App extends React.Component<MainProps, MainState> {
                 exact
                 component={SystemProfiles}
               />
-              <SecureRoute path="/system/new/" component={SystemIntake} />
+              <SecureRoute path="/system/new" component={SystemIntake} />
               <SecureRoute
                 path="/system/:profileId"
                 component={SystemProfile}
               />
-              <SecureRoute path="/business/new/" component={BusinessCase} />
+              <SecureRoute
+                path="/new-system/review/:systemID"
```

Comment:
I'm guessing this is a temporary route?

Reply:
I'm open to suggestions. When I started, I was just putting in `/new-system/review`. When I needed an actual system intake, I added :systemID at the end. I wanted the path to be noticeably different than `/system/new` though, especially since that has a review page in it, too.

Reply:
Maybe `/system/:systemId/grt-review`?